pom.xml

```diff
@@ -165,9 +166,16 @@
             </dependency>
 
             <dependency>
-                <groupId>org.mockito</groupId>
-                <artifactId>mockito-core</artifactId>
-                <version>${mockito.version}</version>
+                <groupId>org.powermock</groupId>
```

Comment:
Does this need to be in the root pom as well as the cms and site ones?

Reply:
The modules pom's do reference the version that is specified in the root pom so it's not actually duplicated even though it's a very similar block. 

(<version>${powermock.version}</version> is only in root)

Reply:
Cool, makes sense.